momepy/shape.py

```diff
@@ -32,12 +32,34 @@
 ]
 
 
+def _form_factor(height, geometry=None, area=None, perimeter=None, volume=None):
```

Comment:
Could `geometry` actually be `None` here or should it be a positional argument?

Reply:
Ah, not at the moment as I use it on line 46. But in theory yes, if you pass all the other values as precomputed arrays. I'll keep it positional for now and revisit during the refactoring.